hiera.yaml

```diff
@@ -0,0 +1,21 @@
+---
+version: 5
+
+defaults:  # Used for any hierarchy level that omits these keys.
+  datadir: data         # This path is relative to hiera.yaml's directory.
+  data_hash: yaml_data  # Use the built-in YAML backend.
+
+hierarchy:
+  - name: "osfamily/major release"
```

Comment:
I think it would make more sense to have an empty hierarchy than the boilerplate. PDK update shouldn't touch this file once it's been created I don't think